CHANGES.rst

```diff
@@ -30,6 +30,9 @@ Minor changes
 - Convert changelog section labels to reStructuredText subheadings for improved Read the Docs navigation. See `Issue 982 <https://github.com/collective/icalendar/issues/982>`_.
 - Move sections in Table of Content of Reference guide.
 - Improve :py:class:`icalendar.prop.vDatetime` documentation. See `Issue #946 <https://github.com/collective/icalendar/issues/946>`_.
+- Fix CLI deprecated `argparse.FileType` usage and remove PendingDeprecationWarning. See `Issue #1005 https://github.com/collective/icalendar/issues/1005`
+- CLI now writes output files using UTF-8 explicitly instead of relying on the
+  system default. See `Issue #1005 https://github.com/collective/icalendar/1005`
```

Comment:
Use proper reStructuredText syntax.
```suggestion
- Fix CLI deprecated :py:class:`argparse.FileType` usage and remove ``PendingDeprecationWarning``. See `Issue #1005 <https://github.com/collective/icalendar/issues/1005>`_.
- CLI now writes output files using UTF-8 explicitly instead of relying on the system default. See `Issue #1005 <https://github.com/collective/icalendar/1005>`_.
```